packages/babel-core/src/config/option-manager.js

```diff
@@ -72,10 +77,14 @@ const ALLOWED_PLUGIN_KEYS = new Set([
   "inherits",
 ]);
 
-export default function manageOptions(opts: {}): {
-  options: Object,
-  passes: Array<Array<[ Plugin, ?{} ]>>,
-}|null {
+export default function manageOptions(
+  opts: {},
+):
```

Comment:
This is an interesting case. I guess it makes sense, but it makes the readability here worse to me at least, but I also don't think there's an obvious alternative.